### PR TITLE
Add stop_on_error

### PR DESCRIPTION
**Pull request by shzhng**
`jupyter_client.KernelClient` already provides the functionality - just need to pass it through

### CODE REVIEW

**Comment by shzhng** [comment]
fixes https://github.com/jupyter/jupyter_kernel_test/issues/31

File: jupyter_kernel_test/__init__.py [hunk 1]
```diff
@@ -57,9 +57,11 @@ def test_kernel_info(self):
 
 
     def execute_helper(self, code, timeout=TIMEOUT,
-                       silent=False, store_history=True):
+                       silent=False, store_history=True,
+                       stop_on_error=True):
         msg_id = self.kc.execute(code=code, silent=silent,
-                                 store_history=store_history)
+                                 store_history=store_history,
+                                 stop_on_error=True)
```
Comment (takluyver) [inline jupyter_kernel_test/__init__.py]: This should be `stop_on_error=stop_on_error` - otherwise it's ignoring the parameter passed in ;-)

Comment (shzhng) [reply jupyter_kernel_test/__init__.py]: woops! thanks for catching my mistake

**Comment by takluyver** [comment]
Thanks! I've pushed a new release to PyPI, so hopefully that helps with the R kernel testing. :-)

**Comment by shzhng** [comment]
thanks so much!

**Pull request merged**